metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala

```diff
@@ -404,12 +422,22 @@ private class BloopPants(
         Some(List(baseDirectory))
       }
 
+    val sourceRoots = {
+      val sourceDirs = sources ++ sourcesGlobs.toSeq.flatten.map(_.directory)
```

Comment:
I believe the following is sufficient
```suggestion
      val sourceDirs = List(baseDirectory)
```